src/styles/utils/reset.scss

```diff
@@ -4,4 +4,9 @@
 
 body {
   margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
This shouldn't be in the reset file, as you're assigning properties instead of resetting them. You should add this to the container that holds your calendar like  `.container` and add it to a separate BEM block